src/main/java/uk/gov/hmcts/darts/arm/service/impl/AbstractArmBatchProcessResponseFiles.java

```diff
@@ -317,6 +325,73 @@ && nonNull(armResponseBatchData.getUploadFileFilenameProcessor())) {
         );
     }
 
+    private void processMultipleInvalidLineFiles(ArmResponseBatchData armResponseBatchData) {
+        try {
+            ExternalObjectDirectoryEntity externalObjectDirectory = getExternalObjectDirectoryEntity(armResponseBatchData.getExternalObjectDirectoryId());
+            var invalidLineRecord1 = armResponseBatchData.getArmResponseInvalidLineRecords().getFirst();
+            var invalidLineRecord2 = armResponseBatchData.getArmResponseInvalidLineRecords().getLast();
```

Comment:
This can be moved into one of the nested if conditions (just above if (if (nonNull(invalidLineRecord1) &&)

As we exclusively use it for that condition (So no point grabbing it if we do not need it)